sip.lwr

```diff
@@ -18,16 +18,19 @@
 #
 
 category: baseline
-configure: 'python configure.py -b $prefix/bin -e $prefix/include/python2.7 -d $prefix/lib/python2.7/site-packages/ -v $prefix/share/sip/ INCDIR=$prefix/include/python2.7'
+configure: 'python configure.py -b $prefix/bin -e $prefix/include/python2.7 -d $prefix/lib/python2.7/dist-packages/
```

Comment:
It seems this line is the only necessary one? And will it still work on older distros? What was the issue this fixes?

Reply:
This was required to get `rfnoc` installed on Ubuntu 16.04. Did not test with other distros.